code/src/store/api/ordersApi.ts

```diff
@@ -0,0 +1,16 @@
+import { appApi } from "@/store/api/appApi";
+import { URLS } from "@/constants/requests";
+import { OrderResponse } from "@/types/order.types";
+import { createUrlPath } from "@/utils/createUrlPath";
+
+type UserId = { id: string };
+
+const ordersApi = appApi.injectEndpoints({
+  endpoints: (build) => ({
+    getOrders: build.query<OrderResponse, UserId>({
+      query: ({ id }) => createUrlPath(URLS.orders.get, id)
```

Comment:
I suggest storing `OrderRequest` instead like following:

```jsx
type OrderRequest = {
    id: string
}
```

You can then store it inside `order.types`

Reply:
Changed